odoo/addons/base/models/ir_http.py

```diff
@@ -487,3 +490,39 @@ def _response_by_status(self, status, headers, content):
             return werkzeug.utils.redirect(content, code=301)
         elif status != 200:
             return request.not_found()
+
+    @classmethod
+    def _get_session_store(cls):
+        """
+        Get the session store from the root session store
+        """
+        return http.root.session_store
+
+    @api.autovacuum
+    def _session_vacuum(self):
+        """
+        Remove all session file unused for some time based
+        on the ir.config_parameter base.unused_session_age_limit
+
+        Or default to one week in seconds.
+        """
+        Param = self.env['ir.config_parameter']
+        session_store = self._get_session_store()
+        week_in_seconds = 60*60*24*7
+
+        try:
+            session_age_limit = Param.sudo().get_param(
+                'base.unused_session_age_limit',
+                week_in_seconds
+            )
+            session_age_limit = int(session_age_limit)
+        except ValueError:
+            session_age_limit = week_in_seconds
+
+        last_week = time.time() - session_age_limit
+
+        session_path = Path(session_store.path)
+
+        for session_file in session_path.iterdir():
+            if session_file.stat().st_mtime < last_week:
+                session_file.unlink(True)
```

Comment:
I had to look back at the documentation to remember what the argument was for, IMO we could use a kwarg here
```suggestion
                session_file.unlink(missing_ok=True)
```

Reply:
Yeah okay, it if makes it more clear.